packages/grafana-prometheus/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "author": "Grafana Labs",
-  "license": "Apache-2.0",
+  "license": "AGPL-3.0-only",
   "name": "@grafana/prometheus",
   "private": true,
```

Comment:
You will need to remove this `private` property before you merge this if you want it to be published.